{{cookiecutter.project_slug}}/Makefile

```diff
@@ -196,10 +196,17 @@ kill: kill-training kill-jupyter kill-tensorboard kill-filebrowser
 setup-local:  ### Install pip requirements locally
 	$(PIP) -r requirements.txt
 
+.PHONY: format
+format:  ### Run static code analysis locally
```

Comment:
Incorrect help message. I assume this is "Automatically format the code" rather than run the analysis.